lang/cs/Org.Apache.REEF.Client/YARN/RESTClient/IYarnRMClient.cs

```diff
@@ -41,9 +41,15 @@ internal interface IYarnRMClient
         Task<NewApplication> CreateNewApplicationAsync();
 
         Task<NewApplication> CreateNewApplicationAsync(CancellationToken cancellationToken);
+
+        Task<Application> SubmitApplicationAsync(SubmitApplication submitApplicationRequest);
+
+        Task<Application> SubmitApplicationAsync(
+            SubmitApplication submitApplicationRequest,
+            CancellationToken cancellationToken);
     }
 
-    [DefaultImplementation(typeof(YarnConfigurationUrlProvider))]
+    [DefaultImplementation(typeof (YarnConfigurationUrlProvider))]
     internal interface IUrlProvider
```

Comment:
It is better to put it in a separate file so that people can easily find the interface from the folder or search by file name.